backends/arm/arm_partitioner.py

```diff
@@ -127,19 +80,13 @@ def partition(self, exported_program: ExportedProgram) -> PartitionResult:
         # subgraphs containing the nodes with the tags
         logger.info("ArmPartitioner::partition")
         partition_tags = {}
-
-        for spec in self.delegation_spec.compile_specs:
-            if spec.key == "quantize_io" and spec.value.decode() == "True":
-                # Exclude IO quantization from the partition
-                passes = PassManager(
-                    passes=[
-                        TagIOQuant(exported_program),
-                    ]
-                )
-                passes(exported_program.graph_module)
+        graph_module = ArmPassManager().transform_partition_pipeline(
+            graph_module=exported_program.graph_module,
+            compile_spec=self.delegation_spec.compile_specs,
+        )
```

Comment:
This is not gonna work. 

The invariant is that input and output of`partitioner()`  should be the same, except a tag.

https://github.com/pytorch/executorch/blob/main/exir/backend/backend_api.py#L368-L371

As @digantdesai mentioned, this should be a user pass before calling `to_backend(ArmPartitioner)` for now. 

We might introduce a first class citizen API for backend specific passes, but for now, we will ask users to add this one-line call to `ArmPassManager()`

Reply:
Functionally nothing has changed since https://github.com/pytorch/executorch/pull/3056. The tag_io_quant_pass has been moved from one file to another. Also, that pass only annotates a tag.

If I understand you correctly, you simply want to remove this call from the partitioner? You don't want the PassManager to be changed?

Reply:
Perhaps we can merge this PR, which just sheds light on the structural issues with quantized IO, and continue the more structural discussions in a new Github ticket?

@oscarandersson8218 @robell @mergennachin @digantdesai